.travis.yml

```diff
@@ -29,11 +29,16 @@ branches:
 
 matrix:
   include:
-    - env: CONFIG=linux_noarch UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
-      os: linux
+    - env: CONFIG=linux_ppc64le_python3.6 UPLOAD_PACKAGES=True DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
+      language: generic
+      os: linux-ppc64le
```

Comment:
just `linux` here

Reply:
Sorry, I had cut and pasted the .travis.yml for the last ray PR. Should be fixed now.